README.md

```diff
@@ -54,6 +54,16 @@ env_variables:
   JETTY_MODULES_ENABLE: 'gzip'
 ```
 
+### Using Quickstart
+Jetty provides [mechanisms](http://www.eclipse.org/jetty/documentation/9.4.x/quickstart-webapp.html) to speed up the start time of your application by pre-scanning its content and generating configuration files.
+If you are using an [extended image](https://github.com/GoogleCloudPlatform/jetty-runtime/blob/master/README.md#extending-the-image) you can activate quickstart by adding this command to your Dockerfile.
+
+```dockerfile
+RUN /docker-entrypoint.bash /scripts/jetty/quickstart.sh
```

Comment:
@cassand I was thinking more along the lines of `RUN /scripts/jetty/quickstart.sh` and having `quickstart.sh` use `docker-entrypoint.bash` if needed. Does that not make sense?